Diagnostics/HealthChecker/Tests/HealthCheckerTest.CommonMocks.NotPublished.ps1

```diff
@@ -96,6 +96,8 @@ Mock Get-RemoteRegistryValue {
         "MinimumConnectionTimeout" { return 0 }
         "LmCompatibilityLevel" { return $null }
         "UBR" { return 720 }
+        "ProductName" { return Import-Clixml "$Script:MockDataCollectionRoot\OS\RemoteRegistryValueProductName.xml" }
+        "InstallationType" { return Import-Clixml "$Script:MockDataCollectionRoot\OS\RemoteRegistryValueInstallationType.xml" }
```

Comment:
Do we need the files? Or can we just provide a return value here instead?

Reply:
I think a return value should work. Will change it.

Reply:
Let's keep the files for now as we have different return values based on the Exchange version / OS (for example, Windows Server 2019 Datacenter and Server Core on Windows Server 2019)

Reply:
Yeah, that is true. I was thinking about this yesterday. Going to need to create a better pester testing setup for this so we don't need to update and add files into Exchange 2013 testing sections any longer. This would be a good example of this.